src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmLevitsky.cpp

```diff
@@ -141,7 +141,7 @@ char MSAConsensusAlgorithmLevitsky::getConsensusChar(const MultipleAlignment& ma
     }
     // find all symbols with freq > threshold, select one with the lowest global freq
     char selectedChar = U2Msa::GAP_CHAR;
-    double selectedGlobalPercentage = 1;
+    double selectedGlobalPercentage = 1.1;
```

Comment:
Could you please keep 1 (100%) and use the following comparison below:
```
if (globalPercentage <= selectedGlobalPercentage) {
```

Reply:
No its not correct, we need to set first found (by alphabet) among the symbols with the same value of frequency to consensus. If you want to keep `double selectedGlobalPercentage = 1;` i can suggest to introduce `bool firstHit` variable.

Reply:
Ok, lets add 'isFirstHit/Match' variable. This way it will be very explicit what is going on and there will be no magic constants.